src/components/Person/Person/Person.jsx

```diff
@@ -0,0 +1,16 @@
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age !== undefined && (
+        <p className="Person__age">{`I am ${person.age}`}</p>
```

Comment:
Use formatting for better reading
```suggestion
        <p className="Person__age">
            {`I am ${person.age}`}
        </p>
```